packages/stripe_android/android/src/main/kotlin/com/flutter/stripe/StripeAubecsDebitPlatformViewFactory.kt

```diff
@@ -16,9 +16,9 @@ class StripeAubecsDebitPlatformViewFactory(
     private val sdkAccessor: () -> StripeSdkModule
 ) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
 
-    override fun create(context: Context, viewId: Int, args: Any?): PlatformView {
+    override fun create(context: Context?, viewId: Int, args: Any?): PlatformView {
         val channel = MethodChannel(flutterPluginBinding.binaryMessenger, "flutter.stripe/aubecs_form_field/${viewId}")
         val creationParams = args as? Map<String?, Any?>?
-        return StripeAubecsDebitPlatformView(context, channel, viewId, creationParams, auBECSDebitFormViewManager, sdkAccessor)
+        return StripeAubecsDebitPlatformView(context!!, channel, viewId, creationParams, auBECSDebitFormViewManager, sdkAccessor)
```

Comment:
@jonasbark  I am not convinced of using `!!` here. When we integrate a fix I think we should handle a nullable context in the platform views itself. What do you think?